src/contract-verification/sourcify/verification.md

```diff
@@ -0,0 +1,5 @@
+# Verification
```

Comment:
This file is a stub and can probably just be mentioned on the 6.1 (src/contract-verification/sourcify/README.md) as a link to the verifier page.

"It's out of the scope of this document to explain this" isn't helpful - if something is out of scope, we should link to resources that do explain it where possible.

Reply:
ok, makes sense, pushed another change